src/js/components/ShareButton.jsx

```diff
@@ -9,19 +17,42 @@ class ShareButton extends React.Component {
 			showing: false
 		}
 	}
-
+	
 	render () {
+
+		// Generate ShareServlet sharing url
+		let metaProps = this.props.meta;
+		if (!metaProps)
+			metaProps = this.props;
+		let params = [];
+		if (metaProps.title)
+			params.push('title=' + escape(metaProps.title));
```

Comment:
Minor point: please use our `encURI` over `escape`, as it supports unicode.

Reply:
Will do!